apps/dashboard/app/views/batch_connect/sessions/connections/_novnc.html.erb

```diff
@@ -6,10 +6,10 @@
 
  <div class="row">
   <div class="col-sm-6">
-   <%= f.number_field(:compression, class: 'custom-range', type: 'range', min: 0, max: 9, value: 6, label: "Compression", help: "0 (low) to 9 (high)") %>
+   <%= f.number_field(:compression, class: 'custom-range', type: 'range', min: 0, max: 9, value: Configuration.novnc_connection_compression.to_i, label: "Compression", help: "0 (low) to 9 (high)") %>
   </div>
   <div class="col-sm-6">
-   <%= f.number_field(:quality, class: 'custom-range', type: 'range', min: 0, max: 9, value: 2, label: "Image Quality", help: "0 (low) to 9 (high)") %>
+   <%= f.number_field(:quality, class: 'custom-range', type: 'range', min: 0, max: 9, value: Configuration.novnc_connection_quality.to_i, label: "Image Quality", help: "0 (low) to 9 (high)") %>
```

Comment:
One last thing - and I kinda hate to nitpick, but should this be `novnc_default_quality` instead?